Codeinwp/neve
# feat: Customizer Search component for Settings

### Summary

Added a new component for the Customizer that allows the user to search through all Customizer sections and panels.

### Will affect the visual aspect of the product

YES

### Screenshots 
Demo:
https://vertis.d.pr/zXX7gg

Preview:
![image](https://user-images.githubusercontent.com/23024731/188234200-a7d25de8-0575-4f03-8cb9-c68513723c5d.png)


### Test instructions

1. Go into Customizer
2. Toggle the search feature on the main page:
![image](https://user-images.githubusercontent.com/23024731/188233803-403b42b3-bdcd-4f17-96d5-2300e3c9c01a.png)
3. Check the search results and the navigation


Closes Codeinwp/neve-pro-addon#1438.
